cli/commands/benchmark-server-timing.mjs

```diff
@@ -348,5 +348,16 @@ function outputResults( opt, results ) {
 		tableData.push( tableRow );
 	}
 
+	// Format the numbers with a consistent number of decimal points.
+	if ( opt.output === 'table' || opt.output === 'md' ) {
+		tableData.forEach( ( row ) => {
+			for ( let i = 1; i < row.length; i++ ) {
+				if ( typeof row[ i ] === 'number' ) {
+					row[ i ] = row[ i ].toFixed( 2 );
+				}
+			}
+		} );
+	}
```

Comment:
Not sure about this:
* Not great to format with a hard-coded precision. 2 is not always enough.
* It's a separate change from the overarching addition of Markdown this PR is doing anyway.

I think it would be better to consider this separately. Enabling formatting makes sense, but it needs to be flexible, so that no precision is "force lost".

Reply:
I'll revert this, but the reason for using 2 specifically is that `benchmark-server-timing` is already rounding to 2 points of precision:

https://github.com/GoogleChromeLabs/wpp-research/blob/661802d86a09510c5a4e54b867668935b3bd0f01/cli/commands/benchmark-server-timing.mjs#L276-L345

Reply:
Removed in 986ef6e